Simplify Hetzner docs (#812)
coreos/fedora-coreos-docs: modules/ROOT/pages/provisioning-hetzner.adoc

hcloud ssh-key create --name "$SSH_KEY_NAME" --public-key "$SSH_PUBKEY"
----
+

> . Launch a server. You can create a server in the web console from the snapshot we just created and skip passing an Ignition file if you just want SSH access. This provides an easy way to test out FCOS without first creating an Ignition config. To use an Ignition configuration, pass it to the VM as user data as below.

Reviewer: ```suggestion
. Launch a server.
  You can create a server using the CLI or the web console from the snapshot we just created.
  You can skip passing an Ignition file if you just want SSH access.
  This provides an easy way to test out FCOS without first creating an Ignition config.
  To use an Ignition configuration, pass it to the VM as user data as below.
```

Reviewer: The should be one sentence per line ideally (it's how asciidocs works best)

DATACENTER="fsn1-dc14"                # See: hcloud datacenter list
TYPE="cx22"                           # See: hcloud server-type list
NAME="fedora-coreos-test"

> IGNITION_CONFIG="./config.ign"        # optional, see prerequisites.

Reviewer: ```suggestion
IGNITION_CONFIG="./config.ign"        # Optional, see prerequisites
```
